src/components/config-close-action.tsx

```
@@ -0,0 +1,67 @@
+import type { AppConfig } from '@/hooks/use-app-config'
+import { Description, ListBox, Select } from '@heroui/react'
+import { useMutation } from '@tanstack/react-query'
+import { invoke } from '@tauri-apps/api/core'
+import { useTranslation } from 'react-i18next'
+import { toast } from '@/utils/toast'
+import { CLOSE_ACTION_OPTIONS, normalizeCloseAction } from '../utils/close-action'
+import { useAppConfig } from '@/hooks/use-app-config'
```

ESLint

[error] Expected "`@/hooks/use-app-config`" (value-internal) to come before "../utils/close-action" (value-parent).

(perfectionist/sort-imports)